internal/server/models/grant.go

```diff
@@ -42,8 +37,6 @@ type Grant struct {
 	Subject   uid.PolymorphicID `validate:"required"` // usually an identity, but could be a role definition
 	Privilege string            `validate:"required"` // role or permission
 	Resource  string            `validate:"required"` // Universal Resource Notation
-
-	CreatedBy uid.ID
```

Comment:
making this a global field so it can be used by all models

Reply:
Reverted this since there were some unexpected side effects such as adding `CreatedBy` to `Settings`